client/reader/following-edit/empty.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import i18n from 'lib/mixins/i18n';
+import { recordAction, recordGaEvent } from 'reader/stats';
+
+const FollowingEditEmptyContent = React.createClass( {
+	recordAction() {
+		recordAction( 'clicked_discover_on_empty' );
```

Comment:
can we add a tracks event? and make this one specific to manage? maybe `clicked_discover_on_empty_manage`